webapp/i18n/en.json

```diff
@@ -188,7 +188,7 @@
   "admin.email.requireVerificationDescription": "Typically set to true in production. When true, Mattermost requires email verification after account creation prior to allowing login. Developers may set this field to false so skip sending verification emails for faster development.",
   "admin.email.requireVerificationTitle": "Require Email Verification: ",
   "admin.email.selfPush": "Manually enter Push Notification Service location",
-  "admin.email.smtpPasswordDescription": " Obtain this credential from administrator setting up your email server.",
+  "admin.email.smtpPasswordDescription": " Obtain this credential from Administrator setting up your email server.",
```

Comment:
Sorry, this is not a change requested in the ticket, and we leave "Administrator" as "administrator" if it's not the context of "Team Administrator" or "System Administrator"?